src/components/BlockDocumentsTable.vue

```diff
@@ -93,31 +95,34 @@
     },
   ])
 
-  const filtered = computed(() => props.blockDocuments.filter(filterBlockDocument))
-
-  function filterBlockDocument({ name, blockType, blockSchema }: BlockDocument): boolean {
-    const { capabilities: blockSchemaCapabilities } = blockSchema
-    const { name: blockTypeName } = blockType
-
-    if (selectedCapability.value && !blockSchemaCapabilities.includes(selectedCapability.value)) {
-      return false
-    }
-
-    if (selectedType.value && blockTypeName != selectedType.value) {
-      return false
-    }
+  const page = useRouteQueryParam('page', NumberRouteParam, 1)
+  const capabilities = ref<string[]>([])
+  const blockTypes = ref<string[]>([])
+  const searchTerm = ref('')
+  const searchTermDebounced = useDebouncedRef(searchTerm, 500)
 
-    return `${name} ${blockType.name} ${blockSchemaCapabilities.join(' ')}`.toLowerCase().includes(searchTerm.value.toLowerCase())
-  }
+  const { filter } = useBlockDocumentsFilterFromRoute({
+    blockSchemas: {
+      blockCapabilities: capabilities,
+    },
+    blockDocuments: {
+      nameLike: searchTermDebounced,
```

Comment:
Does this filters get ignored if the key is included but the value is `null` or `''`?

Reply:
Yes, any falsey value is ignored https://github.com/PrefectHQ/prefect-ui-library/blob/main/src/maps/filters.ts#L60